ext/mbstring/php_mbregex.c

```diff
@@ -465,6 +465,7 @@ static php_mb_regex_t *php_mbregex_compile_pattern(const char *pattern, size_t p
 	OnigEncoding enc = MBREX(current_mbctype);
 
 	if (!php_mb_check_encoding(pattern, patlen, php_mb_regex_get_mbctype_encoding())) {
+		// TODO convert to ValueError after adding pattern_arg_num pass through argument
```

Comment:
This should not be promoted. Incorrectly encoded data is not (necessarily) a programming error.

Reply:
ACK